comp/core/telemetry/telemetryimpl/prom_counter.go

```diff
@@ -3,8 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build !serverless
```

Comment:
I presume you were able to validate the compiler is able to figure this out and there's no trace of `prometheus` in the build despite not having a build tag in these parts of the code?